src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -20,7 +20,7 @@
 use ApiPlatform\Core\Util\ClassInfoTrait;
 use Symfony\Component\PropertyAccess\PropertyAccess;
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
-use Symfony\Component\Routing\Exception\ExceptionInterface;
+use Symfony\Component\Routing\Exception\ExceptionInterface as RoutingExceptionInterface;
```

Comment:
Because `ExceptionInterface` is not specific enough, especially considering we also have our own `ExceptionInterface`.